7-hvs/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  hello-service:
+    image: ${TF_VAR_dockerhub_id}/helloservice:latest
+    container_name: helloservice
+    build: ./HelloService
+    environment:
+      RESPONSE_SERVICE_HOST: responseservice
+    ports:
+      - "9999:5000"
```

Comment:
Are we running this locally?, if not then we can use port 5000 awell.

Reply:
lets keep it same as consul